common/src/main/java/eu/europeana/fulltext/config/AnnoTypes.java

```diff
@@ -0,0 +1,14 @@
+package eu.europeana.fulltext.config;
+
+/**
+ * Created by luthien on 2019-07-10.
```

Comment:
Whoops ... it seems that pressing "Resolve conversation" before clicking on "Add single comment" removes whatever I typed. So I have to type it again. Never a dull moment ...

Anyhow: there is no functional relation with expandDCType. The values in AnnoTypes were used to calculate two meta-annotation-boolean-types _isMedia_ and _isTopLevel_ that I created to make the logic that distinguishes between Annotation.type= 'P' vs Annotation.type = '{something smaller}' generic, so it could be used for the AV-captions as well. Originally, that code was in the XMLParserService class, but I moved it to the Annotation bean class, and forgot to move those variables there, too. I just did that, and this class is now gone.